src/main/java/com/j256/ormlite/misc/functional/Supplier.java

```diff
@@ -0,0 +1,14 @@
+package com.j256.ormlite.misc.functional;
+
+/**
+ * This is a functional interface intended to support lambda expressions in ormlite.
+ */
+public interface Supplier<T> {
+
+    /**
+     * Gets a value.
+     *
+     * @return a value
+     */
+    T get();
```

Comment:
Please rename to be `createEntity()` or something more descriptive than `get()`.